packages/job-worker/src/db/collections.ts

```diff
@@ -92,44 +113,76 @@ export interface IDirectCollections {
 	ExternalMessageQueue: ICollection<ExternalMessageQueueObj>
 }
 
-export function getMongoCollections(client: MongoClient, dbName: string): Readonly<IDirectCollections> {
+export function getMongoCollections(
+	client: MongoClient,
+	dbName: string,
+	allowWatchers: boolean
```

Comment:
This parameter could use some documentation. What is it's purpose?
(same thing with `#allowWatchers` in `WrappedCollection`)

Reply:
better?